src/app/records/PresentationNode.tsx

```diff
@@ -81,6 +83,17 @@ export default function PresentationNode({
 
   const title = <PresentationNodeTitle def={nodeDef} overrideName={overrideName} />;
 
+  // makes shallow copy so original node will be unsorted, makes toggle work
```

Comment:
What does this mean, "makes toggle work"? I buy not modifying the original (it should actually be forbidden because the property is `readonly`) but I don't know what it has to do with the toggle.

Reply:
If I don't make the shallow copy, after clicking the sort button, if I click it again the presentation nodes will stay sorted. I'll try word the comment differently but I did that so the records page can switch between sorted and unsorted when the button is toggled.